### ISSUE TITLE
undefined method `visit'

### ISSUE THREAD

**Issue by JohnSmall** [teamcapybara/capybara#366]
I'm setting up a clean new project in Rails 3, and using capybara instead of webrat. But when I try to get Cucumber to go to the home page then it dies while running the steps. The full message is

undefined method `visit' for #Cucumber::Rails::World:0x3fe3a1a1fcd8 (NoMethodError)

There are a lot of postings about this on various forums, but the solutions all refer to WebRat, not Capybara. So I'm puzzled how I get this error when I've set things up with Capybara.

Basically the claim ;-

> > Capybara is built to work nicely with Cucumber. Support for Capybara is built into cucumber-rails. In your Rails app, just run:

```
rails generate cucumber:install --capybara
```

And everything should be set up and ready to go.

> > Is false, because it isn't set up and ready to go. There's a secret step known only to the initiated. Could someone please tell me what it is.
